content/fr/user/signup.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Créer un compte
+description: Trouver votre communauté idéale.
+menu:
+  docs:
+    weight: 10
+    parent: user
+---
+
+## Choisir une instance {#picker}
+
+Vous devez choisir une instance pour vous y inscrire, comme vous choisiriez un fournisseur d’email, ou un royaume _World of Warcraft_ pour votre personnage. Le site web sera votre fournisseur de service, hébergeant votre compte, votre profil, et vore fil d’actualité local.
+
+{{< hint style="info" >}}
+Vous pouvez [parcourir une liste de serveurs par catégories et par langues sur joinmastodon.org](https://joinmastodon.org/#getting-started).
+{{< /hint >}}
+
+### Comprendre la politique d’une instance {#tos}
+
+Avant de vous inscrire à un service, il est important d’en comprendre la politique et les conditions d’utilisation. La politique d’une instance Mastodon se trouve généralement sur la page `/about/more`, qui se trouve en cliquant « À propos  » sur la page d’accueil de cette instance, en étant déconnecté.
+
+### Types d’inscriptions {#signup}
+
+Mastodon permet aux personnes qui administrent une instance de définir un mode d’inscription parmi trois : inscriptions libres, sur invitation, et par approbation. 
+
+#### Inscriptions libres {#open}
+
+Certaines instances vous permettront de vous enregistrer immédiatement —validez simplement le formulaire d’inscription avec votre nom d’utilisateur, adresse de courriel, et mot de passe, et vous pourrez utiliser votre compte.
+
+#### Sur invitation {#invite}
+
+Certaines instances désactivent le formulaire d’inscription, et demandent à la place que des liens d’invitation soient générés et partagés pour permettre aux gens de s’inscrire.
+
+#### Inscription par approbation {#approval}
+
+Certaines instances vous autorisent à remplir un formulaire d’inscription, mais avec un champ de formulaire supplémentaire pour expliquer pourquoi vous souhaitez rejoindre cette instance. Une fois le formulaire validé, votre compte doit être approuvé par un membre de l’équipe de modération avant de pouvoir l’utiliser. 
+
+## Votre nom d’utilisateur et votre domaine {#address}
+
+<!-- TODO: Replace username by address in the English version. -->
+
+Les adresses Mastodon se composent de deux parties :
+
+* Le nom local, e.g. `claude`
```

Comment:
Remarque de goût personnel : je préfère Camille, plus moderne :)

Reply:
Let’s assume we’re trying to convince older folks and the younger generation won’t mind. :p